modules/tensor_mechanics/test/tests/rom_stress_update/gold/3tile_json_export.json

```diff
@@ -1 +0,0 @@
-../../../../data/laromance/test/3tile.json
```

Comment:
Please keep this a symbolic link!

Reply:
Yeah, this shouldn't be in here...try
```
cd modules/tensor_mechanics/test/tests/rom_stress_update/gold/
rm ./3tile_json_out.csv
ln -s ./3tile_out.csv ./3tile_json_out.csv
```

Reply:
So looks like this file got changed in the shuffle too, and should remain a symbolic link? I don't think you need to change it, correct?

Reply:
This is still not fixed in the latest commit 13mins ago!